### ISSUE TITLE
Updated container notifications not working when --run-once is used

### ISSUE THREAD

**Issue by regystro** [containrrr/watchtower#603]
Hi. When using --run-once instead of scheduling mode, the only notification I receive is "Running a one time update.", even when new images are pulled and containers are correctly updated (testing for a week and checking with image and container recreation date/time). 

This is my command line:
```
docker run --rm --name watchtower \
 -v /var/run/docker.sock:/var/run/docker.sock \
 -v /etc/timezone:/etc/timezone:ro \
 containrrr/watchtower \
 --run-once --cleanup --debug \
 --notifications shoutrrr \
 --notification-url "telegram://<my_telegram_token>@telegram?channels=<my_telegram_channel>"
```

**Expected behavior**
When running the same command without the --run-once flag (scheduled mode), I get the correct notifications:
```
Starting Watchtower and scheduling first run: 2020-07-16 16:10:38 +0000 UTC m=+1802.529196971

Found new linuxserver/jackett:latest image (sha256:ffef8c6c4b8228bc96b2c73d5676ee1a08be1937d33fc3c2a8fa9f9be1058872)
Stopping /jackett (b39e1ff3309316da7abbd6adc08635adfe52d7b72372a101ee65111842ee4a40) with SIGTERM
Creating /jackett
Removing image sha256:b3b2ea1fcc423136ae70bd41fa0b3a438a70f8498cccb176ad5a89a6fc88d89f
```

**Environment**
 - Platform: Raspberry Pi
 - Architecture: armv7l
 - Docker version: 19.03.12, build 48a6621

**Logs from running watchtower with the `--debug` option**
These are obtained when using --run-once but no updates were found. When there are updates, they appear in the logs, but there's no Telegram notification about them:
```
time="2020-07-17T08:00:17Z" level=info msg="Running a one time update."
time="2020-07-17T08:00:17Z" level=debug msg="Checking containers for updated images"
time="2020-07-17T08:00:17Z" level=debug msg="Retrieving running containers"
time="2020-07-17T08:00:19Z" level=debug msg="Pulling containrrr/watchtower:latest for /watchtower"
time="2020-07-17T08:00:19Z" level=debug msg="No credentials for containrrr in /config.json"
time="2020-07-17T08:00:19Z" level=debug msg="Got image name: containrrr/watchtower:latest"
time="2020-07-17T08:00:21Z" level=debug msg="No new images found for /watchtower"
time="2020-07-17T08:00:21Z" level=debug msg="Pulling linuxserver/jackett:latest for /jackett"
time="2020-07-17T08:00:21Z" level=debug msg="No credentials for linuxserver in /config.json"
time="2020-07-17T08:00:21Z" level=debug msg="Got image name: linuxserver/jackett:latest"
time="2020-07-17T08:00:23Z" level=debug msg="No new images found for /jackett"
...
time="2020-07-17T08:00:32Z" level=debug msg="This is the watchtower container /watchtower"
```

Thank you.

**Comment by piksel** [comment]
Should be fixed by #564

**Comment by regystro** [comment]
OK, thanks. Will the fix be available in the next release? 🙏

**Comment by simonaronsson** [comment]
> OK, thanks. Will the fix be available in the next release? 🙏

Yes 👍

**Comment by regystro** [comment]
I can confirm this issue is fixed in v1.0.3.

Thanks a lot.
